sql/src/main/java/io/crate/action/sql/RowReceiverToResultReceiver.java

```diff
@@ -27,7 +27,7 @@
 
 import java.util.Set;
 
-class RowReceiverToResultReceiver implements RowReceiver {
+public class RowReceiverToResultReceiver implements RowReceiver {
```

Comment:
The thing is that it's used in the protocols.postgres and in action.sql
Any suggestions?